.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+---
+name: Aexpect CI
+
+on:
+  pull_request:
+  push:
+  workflow_dispatch:
+
+jobs:
+  selftests:
+    name: ${{ matrix.python-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.x]
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - run: pip install -r requirements-travis.txt
+      - run: make check
```

Comment:
You can put this two actions in the same run command. Or instead adding a name to the step so it's easier to read in the web, like:

```
- name: installing requeriments
  run: pip install -r requirements-travis.txt
- name: run checks
  run: make check
```

Reply:
I know but these one-liners are self-explanatory enough. If there were many steps than bundling them together would make sense but putting a name to "pip install" or "make check" seems unnecessary to me. Anyway I can change that if you think otherwise...

Reply:
Current lines are correct, this is a matter of taste.